README.md

```diff
@@ -51,3 +51,21 @@ Code coverage will be generated on path:
 ```
 {project-root}/{module}/target/scala-{scala_version}/jacoco/report/html
 ```
+## Using Jconsole to monitor /actuator/health
```

Comment:
Consider restructuring of the section. I suggest:
## Health check endpoint
[a sentence or two that there is SpringBoot Actuator for Health implementation, reachable via HTTP and JMX]
### via HTTP endpoint 
[mention that the HC endpoint can be found in at `http(s)://server:port/actuator/health`, also in swagger UI - see other comment]
### via JMX 
[Explain that only local-process JMX binding is enabled, remote is not but could be via config...]
#### Connecting to local JMX endpoint - steps
[here you can put the steps you have written]